src/less/carousel/carousel.less

```diff
@@ -24,12 +24,12 @@
         right: 0;
     }
 
-    .icon--chevron-right-24 {
```

Comment:
Should we keep the old styles so its not breaking?

Reply:
That's a good point, we should keep this and maybe leave a `/* deprecated */` comment

Reply:
Ohhhh, good catch Andrew. My rustiness is showing through! Will fix.